tests/e2e/comments.spec.js

```diff
@@ -1,36 +1,28 @@
-const {test, expect} = require('@playwright/test');
-import {login} from './tools/lib/login';
-import {rest} from './tools/lib/rest';
+const {test, expect} = require('./tools/lib/test-utils');
 
-test('Test adding a Comment to a Post', async ({page, context}) => {
-  test.setTimeout(240 * 1000);
-  await page.goto('./');
-  await login(context);
+test.useAdminLoggedIn();
 
+test('Test adding a Comment to a Post', async ({page, requestUtils, admin}) => {
   // Post creation using rest api since we already have a test for Post creation
-  const newPost = await rest(context, {
-    path: './wp-json/wp/v2/posts',
-    method: 'POST',
-    data: {
-      title: 'Test Post',
-      content: `
-        <!-- wp:paragraph -->
-          <p>This is a test post</p>
-        <!-- /wp:paragraph -->
-      `,
-      status: 'publish',
-    },
+  const newPost = await requestUtils.createPost({
+    title: 'Test Post',
+    content: `
+      <!-- wp:paragraph -->
+        <p>This is a test post</p>
+      <!-- /wp:paragraph -->
+    `,
+    status: 'publish',
   });
 
   await page.goto(newPost.link);
-  const commentTextbox = page.getByPlaceholder('Your Comment');
+  const commentTextbox = await page.getByPlaceholder('Your Comment');
```

Comment:
According to VS Code we don't need `await` here:

<img width="660" alt="Screenshot 2023-07-12 at 10 02 34" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/c71bdfc2-0d72-41ab-949d-ea69ab76e941">

Reply:
Correct, but you need it for the click & fill part below.

Just `await page.getByPlaceholder('Your Comment').fill('Nice Post')` should work though.